### ISSUE TITLE
Evaluate any accessibility linter tools

### ISSUE THREAD

**Issue by brenoferreira** [creativecommons/cccatalog-frontend#1000]
Evaluate how an accessibility linter tool could be added to our CI build pipeline, so that we don't let our accessibility score go down with time.

To do this, it will be necessary to start a dev server and run the accessibility tool with a [headless browser](https://developers.google.com/web/updates/2017/04/headless-chrome) and read the accessibility score from the CLI output and compare with the current score on the main branches (`develop` or `master`, depending on which is the target build branch) to check if it's equal or higher. If it's lower, the build should fail.

Useful links:

* Lighthouse CLI https://github.com/GoogleChrome/lighthouse#using-the-node-cli
* Headless Chrome: https://developers.google.com/web/updates/2017/04/headless-chrome
* pa11y ESLint plugin: https://www.npmjs.com/package/eslint-plugin-jsx-a11y

**Comment by AyanChoudhary** [comment]
this still needs work to be done upon

**Comment by AyanChoudhary** [comment]
@zackkrida I found this [tool](https://github.com/GoogleChrome/lighthouse-ci) for lighthouse ci which provides inbuilt features for comparing scores against another version of the site and matches our use case.

**Comment by zackkrida** [comment]
Closing as we will implement [lighthouse ci](https://github.com/GoogleChrome/lighthouse-ci) in the future for this
